jenkinsapi/jenkins.py

```diff
@@ -1,15 +1,18 @@
 """
 Module for jenkinsapi Jenkins object
 """
+
```

Comment:
The Jenkins object can be seen as a dict of jobs and has other dicts in it: Views, Slaves.  Those child dicts has methods, that are directly relevant to their particular area. So the code that deals with getting stuff from update center is better to be part of Plugins class. I find that a bit cleaner than ever growing Jenkins object.

At the same time Jenkins class has some methods that are left for backwards compatibility with previous versions. For example: it has `get_jobs()` which is calling `Jobs` object anyway - removal that method will certainly break a lot of existing scripts that use jenkinsapi.